Improve docs around wallet permissions (#1026)
MetaMask/metamask-docs: wallet/how-to/manage-permissions.md

    </div>
</div>

:::info Accessing accounts

> We recommend using [`eth_requestAccounts`](/wallet/reference/eth_requestAccounts), which

Reviewer: Does this change make it easier for the user to understand?

"We suggest using eth_requestAccounts, which automatically asks for permission to use eth_accounts by invoking wallet_requestPermissions internally."

      }
    });
}
```

## Revoke permissions example

> The following is an example of using `wallet_revokePermissions` to revoke the dapp's permission to
> call `eth_accounts`:

Reviewer: ```suggestion
The following example uses `wallet_revokePermissions` to revoke the dapp's permission to
call `eth_accounts`:
```
